utils/telldus-core/Makefile

```diff
@@ -23,6 +23,18 @@ PKG_BUILD_DEPENDS:=argp-standalone
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk
 
+define Download/argp-cmake
```

Comment:
This downloads Findargp.cmake into dl/ . Not a big deal but maybe would be better to download into the download directory only. Not sure how to do that.